src/framework/mlt_properties.c

```diff
@@ -2246,6 +2246,19 @@ void mlt_properties_clear( mlt_properties self, const char *name )
 	mlt_events_fire( self, "property-changed", name, NULL );
 }
 
+/** Check if a property exists.
```

Comment:
Today, much of the code in MLT and elsewhere simply does `mlt_properties_get()` to check for existence, but that may invoke conversion or serialization of the property. Are you specifically trying to avoid those side effects or forgot the convention?

Reply:
mlt_properties_get() will break an animation and replace it with a string representation of the last value in the animation. I suppose this change is actually working around a bug. But I also think that it adds clarity to the code. I had been wishing for such a function in the past but never took the time.

Reply:
I am OK to add `mlt_properties_exist()` but it should be added to Mlt::Properties as well. Due to heavy inheritance, the method on it should be named like `property_exists()` so it does not read like `filter.exists(..)`, which can be misleading .